script_library/deployment-actions-openstack.sh

```diff
@@ -32,7 +32,7 @@ function deploy_caasp4(){
     source ${scripts_absolute_dir}/pre-flight-checks.sh check_openstack_environment_is_ready_for_deploy
     source ${scripts_absolute_dir}/pre-flight-checks.sh check_caasp4_skuba_available
     source ${scripts_absolute_dir}/pre-flight-checks.sh check_caasp4_terraform_available
```

Comment:
Also the terraform needs that the variable TERRAFORM_BINARY_PATH exists and not empty. If we don't check it this check stucks forever here

This is a possible solution
check_caasp4_terraform_available(){
    echo "Checking for CaaSP 4 that terraform is available"
    if [ -z ${TERRAFORM_BINARY_PATH+x} ]; then
        echo "Terraform is not installed. Can not deploy CaaSP 4"
        exit
    fi
    ... Following lines
}

I added it to the PR I am preparing for the card SOC-9927 or we can include it here

Reply:
I did not touch that, its coming from master so not touching it further.

Please send the PR into master if thats an issue :)

Reply:
TERRAFORM_BINARY_PATH is always set when you use run.sh . Or how do you run into the problem that the var does not exist?

Reply:
In any case, it is imo unrelated to this PR...